browser/src/DataPage/exampleRawGraphQLQuery.ts

```diff
@@ -0,0 +1,126 @@
+const content = `# Lines beginning with a "#" character are comments that are ignored by the API.
+#
+# "VariantsInGene" is the operation name for the query. This is optional but
+# recommended.
+#
+query VariantsInGene {
+  # We could alternatively look up the gene by Ensembl ID by using the
+  # "gene_id" argument rather than "gene_symbol".
+  #
+  # The gene symbol is given with quotes, but not the reference genome or
+  # dataset. This is because the gene symbol is defined in the API schema as a
+  # string, but the reference genome is an enumerated type that can have only
+  # the value GRCh37 or GRCh38, and the dataset argument is constrained
+  # similarly. See
```

Comment:
minor rewording:
In this query, the gene symbol must be provided with quotes ("BRCA1"). This does not apply to the reference genome build or gnomAD dataset version because the gene symbol is defined as a string, but the reference genome and dataset version are defined as enumerated types that only have specific values (e.g., reference genome can only be one of GRCh37 or GRCh38).

open to additional rewordings on top of this as well!